tests/test_py_utils.py

```diff
@@ -1,18 +1,35 @@
+from pickle import loads
```

Comment:
We use `dill` for pickling:
```suggestion
from dill import loads
```